src/reformat_libyuv.c

```diff
@@ -801,12 +1033,35 @@ avifResult avifImageYUVToRGBLibYUV(const avifImage * image, avifRGBImage * rgb)
                                               matrix,
                                               image->width,
                                               image->height);
+        } else if (lcf.yuvaToRgbMatrix != NULL) {
+            libyuvResult = lcf.yuvaToRgbMatrix(image->yuvPlanes[AVIF_CHAN_Y],
+                                               image->yuvRowBytes[AVIF_CHAN_Y],
+                                               image->yuvPlanes[uPlaneIndex],
+                                               image->yuvRowBytes[uPlaneIndex],
+                                               image->yuvPlanes[vPlaneIndex],
+                                               image->yuvRowBytes[vPlaneIndex],
+                                               image->alphaPlane,
+                                               image->alphaRowBytes,
+                                               rgb->pixels,
+                                               rgb->rowBytes,
+                                               matrix,
+                                               image->width,
+                                               image->height,
+                                               /*attenuate=*/0);
+            *alphaReformattedWithLibYUV = AVIF_TRUE;
         }
         if (inputIsHighBitDepth) {
             avifImageFreePlanes(&image8, AVIF_PLANES_YUV);
         }
     }
-    return (libyuvResult != 0) ? AVIF_RESULT_REFORMAT_FAILED : AVIF_RESULT_OK;
+    if (libyuvResult != 0) {
+        return AVIF_RESULT_REFORMAT_FAILED;
+    }
+    if (!image->alphaPlane || !image->alphaRowBytes) {
+        // If the image does not have an alpha plane, then libyuv always prefills the alpha plane with opaque values.
+        *alphaReformattedWithLibYUV = AVIF_TRUE;
+    }
+    return AVIF_RESULT_OK;
```

Comment:
yeah it doesn't matter either way. the newer code looks better since it doesn't do any extra work i.e. there is no benefit to going out of the way to match the existing code. so i'll leave it as-is.